index.js

```diff
@@ -1,124 +1,134 @@
 'use strict';
 
-var cache = Object.create(null);
-var debug = false;
-var hitCount = 0;
-var missCount = 0;
-var size = 0;
-
-exports.put = function(key, value, time, timeoutCallback) {
-  if (debug) {
-    console.log('caching: %s = %j (@%s)', key, value, time);
+var Cache = (function () {
+  var cache = Object.create(null);
+  var hitCount = 0;
+  var missCount = 0;
+  var size = 0;
+
+  function Cache (debug) {
+    this._debug = debug || false;
```

Comment:
@ptarjan I thought it would be easier to use the debug feature.
Instead of call the setter explicitly, you can just add `true`, when you create the instance.